lib/clientstate-normalizer.ts

```diff
@@ -1030,7 +1025,6 @@ export class ClientStateGoldenifier extends GoldenLayoutComponents {
                         tool.id,
                         tool.args,
                         tool.stdin,
-                        width,
```

Comment:
are you sure this is correct? Although it seems links with tools are currently broken, so it's hard to tell...

Reply:
Ah thank you, I think I did make a mistake here. I'm not 100% confident on anything here but going off the parameter names it should probably be
```ts
                    const stack = this.newToolStackFromCompiler(
                        session,
                        idxCompiler + 1,
                        tool.id,
                        tool.args,
                        tool.stdin,
                        width,
                    );
```

Reply:
ohhh that might be why https://github.com/compiler-explorer/compiler-explorer/issues/4726 is broken !